packages/terra-content-container/lib/ContentContainer.scss

```diff
@@ -0,0 +1,37 @@
+.terra-ContentContainer {
+  display: flex;
+  flex-direction: column;
+
+  > .terra-ContentContainer-header {
```

Comment:
Since all of our css classes are namespaced, we should be able to get rid of the descendent selectors.